### DISCUSSION TITLE
how edit the model on height value

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#5060] [Q&A]
now i run the test-apps\display-test-app  
and edit the project location by editor project geolocation move
but can not move the project on height value

**Comment by pmconne** [comment]
You probably want to use [editor project geolocation point](https://github.com/iTwin/itwinjs-core/blob/master/editor/frontend/README.md#adding-or-updating-geolocation) instead and specify an altitude (height).
A "real" app would display tool settings as illustrated in the README linked above where you could interactively input your altitude.
display-test-app does not display tool settings, so you'd need to supply the altitude as a key-in argument.
